.circleci/config.yml

```diff
@@ -107,26 +120,60 @@ jobs:
       - run: sudo make -C build install
       - run: make -C build package
 
+  build-arm-release:
+    docker:
+      - image: arm64v8/ubuntu:23.04
+    resource_class: arm.xlarge
+    steps:
+      - checkout
+      - run: apt update && apt install sudo
```

Comment:
this is one main difference, the basic ubuntu image doesnt come with sudo preinstalled